pom.xml

```diff
@@ -21,6 +21,7 @@
     </properties>
 
     <dependencies>
+<!--        spring core-->
```

Comment:
let's remove comments from pom
they can be helpful during developing, but no comments should be in production-ready code

Reply:
ok